content/docs/command-reference/cache/dir.md

```diff
@@ -72,4 +73,7 @@ $ cat .dvc/config
 ...
 ```
 
-Absolute path `/path/to/dir` saved as is.
+As compared to the `dvc config cache.dir` command, this is a higher level
+command which is specific to a set of use cases. To know more about usage of
+`dvc config cache` refer to the [cache](/doc/command-reference/config#cache)
+section in the `config` docs.
```

Comment:
Ok, this paragraph is correct but not needed in this example (this difference is already explained in the description), please revert and see my comments above. Thanks